src/shared/logs-show.c

```diff
@@ -491,9 +491,15 @@ static int output_short(
                 return 0;
         }
 
+        if (identifier && set_contains(j->exclude, identifier))
```

Comment:
Is it not necessary to applied to other output functions, e.g. `output_json()`?

Reply:
The full list of output functions are: 
```
output_cat     -- this feature not needed if "cat"
output_export  -- this feature not needed if "export"
output_json    -- typically json output is used for parsing/scripting, so I believe it's not needed for this as well
output_short   -- this feature covers all the variants of output_short
output_verbose -- this feature not needed if "verbose"
```